src/Routes/ListRoutesCommand.php

```diff
@@ -198,7 +187,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return $result;
     }
 
-    public function getRows(bool $requireNames = false): array
+    private function getRows(bool $requireNames = false): array
```

Comment:
```suggestion
    private function getRows(RouteFilterOptions $options, bool $requireNames = false): array
```

Passing filter options as an argument negates the need for a private property. `getRows()` can be changed to suit.

Also, does the table renderer care if the array is a hash or a list? If it doesn't make any difference, you can just always return the hash from `getRows()`, reducing complexity, and drop the `$requireNames` argument